src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -66,6 +66,20 @@
             -n domain-name
 """
 
+helps['cdn custom-domain enable-https'] = """
+type: command
+short-summary: Enable HTTPS delivery of the custom domain.
+examples:
+  - name: Enable HTTPS using a CDN-managed certificate
+    text: >
+        az cdn custom-domain enable-https -g group --profile-name profile --endpoint-name endpoint \\
```

Comment:
You can remove `\\` to let CLI folder it automatically since you are using `>` here to indicate YAML block. Othersize the newline wrapper `\\` might behave unexpected based on different terminal windows size.

Reply:
Fixed.